pkg/cri/sbserver/container_create_linux_test.go

```diff
@@ -42,9 +42,7 @@ import (
 	"github.com/containerd/containerd/pkg/cap"
 	"github.com/containerd/containerd/pkg/cri/annotations"
 	"github.com/containerd/containerd/pkg/cri/config"
-	"github.com/containerd/containerd/pkg/cri/opts"
 	customopts "github.com/containerd/containerd/pkg/cri/opts"
-	"github.com/containerd/containerd/pkg/cri/util"
```

Comment:
These two (`opts` and `util`) seem not have conflict imports. Maybe we should keep the ones without aliases.